pkg/api/testapi/testapi.go

```diff
@@ -106,31 +105,26 @@ func init() {
 			}
 
 			internalGroupVersion := unversioned.GroupVersion{Group: groupVersion.Group, Version: runtime.APIVersionInternal}
-			if group, ok := Groups[groupVersion.Group]; !ok {
-				Groups[groupVersion.Group] = TestGroup{
-					externalGroupVersions: []unversioned.GroupVersion{groupVersion},
-					internalGroupVersion:  internalGroupVersion,
-					internalTypes:         api.Scheme.KnownTypes(internalGroupVersion),
-				}
-			} else {
-				group.externalGroupVersions = append(group.externalGroupVersions, groupVersion)
-				Groups[groupVersion.Group] = group
+			Groups[groupVersion.Group] = TestGroup{
+				externalGroupVersion: groupVersion,
```

Comment:
@soltysh I think this means we only test the last version of a group. Could you leave a note in test-go.sh and test-integration.sh?